ros2topic/ros2topic/verb/pub.py

```diff
@@ -107,6 +116,9 @@ def timer_callback():
         count += 1
         if print_nth and count % print_nth == 0:
             print('publishing #%d: %r\n' % (count, msg))
+        curr_rostime = _clock.now().nanoseconds
+        msg.header.stamp.sec = int(curr_rostime * 1e-9)
+        msg.header.stamp.nanosec = int(curr_rostime % 1e9)
```

Comment:
To avoid doing this "calculation" I added a new method `seconds_nanoseconds` on the `Time` class in #235. Please use that one:

```
msg.header.stamp.sec, msg.header.stamp.nanosec = clock.now().seconds_nanoseconds
```

Reply:
I would think you'd use [`now().to_msg()`](https://github.com/ros2/rclpy/pull/235/files#diff-cf0da220e7cfb92120fc2d55444635c4R129) here instead